### DISCUSSION TITLE
Creating Relations in Sequel Ace

### DISCUSSION THREAD

**Discussion by finestjava** [Sequel-Ace/Sequel-Ace#1304] [Q&A]
OK, I have tried all the variations I could think of and all I get is Foreign Key issues.
Starting simple - Two Tables - table one has 
Table 1
Row A - Primary Key
Row B
Row C
Row D

Table 2
Row A
Row Q
Row C

I want to match up Row A - To show Row C in Table 2

Thanks,
Screen Shots would be a Big Plus

**Comment by samuelbradshaw** [comment]
Make sure the two fields that are being connected (Primary key: Table1.RowA, and Foreign key: Table2.RowC) have the same data type (both INTs, for example). If it's an INT field, make sure that both fields are unsigned or signed (for a key, you probably want unsigned, but what matters is that they're the same). This has thrown me off in the past.

**Reply by Jason-Morcos** [reply]
@finestjava Can you share the create statements for the tables if you're still having issues after @samuelbradshaw's good suggestion?
